doc/nrf/gs_installing.rst

```diff
@@ -11,10 +10,586 @@ See the :ref:`gs_assistant` section for information about how to install the Get
    If you use the Getting Started Assistant, you can skip this section of the documentation.
    If you prefer to install the toolchain manually, or if you run into problems during the installation process, see the following documentation for instructions.
 
+To manually install the |NCS|, you must ensure that all required tools are installed and clone the |NCS| repositories.
+See the following sections for detailed instructions.
+
+The first two steps, `Installing the required tools`_ and `Installing the toolchain`_, are identical to the installation in Zephyr.
+If you already have your system set up to work with the Zephyr OS, you can skip these steps.
+
```

Comment:
Suggesting to move this here instead of its current location(towards the end)  in the document.
````suggestion
See :ref:`gs_installing_os` for information on the supported operating systems and Zephyr features.
````